app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  # Method to update the comments counter for a post
+  after_create :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.increment(:comments_counter)
+  end
```

Comment:
- [ ] It would be nicer to call the update_post_comments_counter every time a post is created and saved to the database. Make sure the` after_save` method is called just on top after the validates. And make the update_post_counter method private.
``` 
 after_save :update_post_comments_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```
If you use after_save, the update_post_counter method will be called after both creating a new record and updating an existing one but If you use after_create, the update_post_counter method will only be called after creating a new record.